index.js

```diff
@@ -151,11 +150,11 @@ module.exports = {
                     if (session) {
                         var filename = 'ScreenShot_onException-' + process.pid + '-' + new Date().getTime();
                         var screenShotFileName = path.resolve(screenShotPath, filename);
-                        flow.wait(function () {
-                            return nemo.screenshot.snap(screenShotFileName).then(function () {
-                                exception._nemoScreenshotHandled = true;
-                                throw exception;
-                            }, 10000);
+                        nemo.screenshot.snap(screenShotFileName).then(function () {
```

Comment:
So, the original code would re-fail if 10 seconds passed and the promise had not resolved. Is there a possibility that any new hangs could arise as a result of not having that promise kill in there?